src/reaper_osara.cpp

```diff
@@ -4784,17 +4784,18 @@ void cmdTransientDetectionSettings(Command* command) {
 	// We must register the hook before the dialog appears or it won't work.
 	plugin_register("accelerator", &transDetect_accelReg);
 	// Open the dialog.
-	Main_OnCommand(command->gaccel.accel.cmd, 0);
+	
+Main_OnCommand(command->gaccel.accel.cmd, 0);
```

Comment:
I think this line accidentally got split.

```suggestion
	Main_OnCommand(command->gaccel.accel.cmd, 0);
```